src/Umbraco.Web/ContentApps/ListViewContentAppFactory.cs

```diff
@@ -99,6 +99,11 @@ public ContentApp GetContentAppFor(object o, IEnumerable<IReadOnlyUserGroup> use
                 if (configTabName != null && String.IsNullOrWhiteSpace(configTabName.ToString()) == false)
                     contentApp.Name = configTabName.ToString();
             }
+            // if the list view is configured to show umbContent first, update the list view content app weight accordingly
+            if(listViewConfig.ContainsKey("showContentFirst") && listViewConfig["showContentFirst"]?.ToString() == "True")
+            {
```

Comment:
@kjac won't it be simpler to parse the value to a boolean, when the field is defined as boolean? E.g.

```
bool r = bool.TryParse(valueToConvert, out bool b) ? b : false;
```

Reply:
It would possibly be conceptually clearer to do, but the value is stored as a string (hence the capital T), so this is faster.